src/test/java/com/dpgil/pathlinker/path_linker/internal/PathLinkerModelTest.java

```diff
@@ -435,7 +438,7 @@ public void testTargetSourceIdentical () throws IOException {
 		//create the model for algorithm with source = target and k = 37
 		//k value is set to enumerate all paths of length to 0.5329432500000001 ensure the results will be same,
 		//otherwise results could be correct but different due to the random nature of which paths PathLinker finds first
-		testModel = new PathLinkerModel(originalNetworkDir, true, false, 
+		testModel = new PathLinkerModel(originalNetworkDir, true, true, false, 
```

Comment:
instead of passing the boolean values true, true and false, can you pass the global variables you have set for these values?